src/components/Accordion/components/Compact.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState, ReactNode } from 'react';
+import styled from 'styled-components';
+
+import { Colors } from '../../../essentials';
+import { Box } from '../../Box/Box';
+import { Headline } from '../../Headline/Headline';
+import { Header } from './Header';
+import { ChevronUp } from './ChevronUp';
+import { ChevronDown } from './ChevronDown';
+import { Description } from './Description';
+import { PanelButton } from './PanelButton';
+
+interface Props {
+    title: string;
+    description?: string;
+    expanded?: boolean;
+    children?: ReactNode;
+}
+
+const PanelHeader = styled(Header).attrs({ px: '0' })`
+    // @ts-ignore
+    &:hover ${Headline} {
+        color: ${Colors.ACTION_BLUE_1000};
+    }
+
+    &:hover ${ChevronDown} {
+        color: ${Colors.ACTION_BLUE_1000};
+    }
+
+    &:hover ${ChevronUp} {
+        color: ${Colors.ACTION_BLUE_1000};
+    }
+`;
+
+const Compact = ({ title, description, expanded = false, children }: Props) => {
```

Comment:
To include the official children, you can use the `PropsWithChildren<T>` from the react library.
```suggestion
const Compact = ({ title, description, expanded = false, children }: PropsWithChildren<Props>) => {
```

Reply:
Done